app-5-rdf-serializer/src/main/java/org/geovistory/toolbox/streams/rdf/RegisterInputTopic.java

```diff
@@ -66,4 +66,13 @@ public KStream<ProjectEntityKey, ProjectEntityLabelValue> projectEntityLabelStre
         );
     }
 
+    public KStream<ProjectRdfKey, ProjectRdfValue> projectStream() {
```

Comment:
The input stream is KStream<dev.projects.project.Key, dev.projects.project.Value>

Also update the serdes below.

Reply:
Done